docs/commands/bundle.md

```diff
@@ -24,27 +24,26 @@ redocly bundle --version
---config | string | Specify path to the [config file](#custom-configuration-file).
---dereferenced, -d | boolean | Generate fully dereferenced bundle.
---ext | string | Specify bundled file extension. Possible values are `json`, `yaml`, or `yml`. Default value is `yaml`.
---extends | [string] | Can be used in combination with `--lint` to [extend a specific configuration](./lint.md#extend-configuration).  Default values are taken from the Redocly configuration file.
---force, -f | boolean | Generate bundle output even when errors occur.
---format | string | Format for the output. Possible values are `codeframe`, `stylish`, `json`, or `checkstyle`. Default value is `codeframe`.
---help | boolean | Show help.
---keep-url-references, -k | boolean | Keep absolute url references.
---lint | boolean | Lint definition files. Default value is `false`.
---max-problems | integer | Truncate output to display the specified maximum number of problems. Default value is `100`.
---metafile | string | Path for the bundle metadata file.
---output, -o | string | Name or folder for the bundle file. If you don't specify the file extension, `.yaml` is used by default. If the specified folder doesn't exist, it's created automatically. **If the file specified as the bundler's output already exists, it's overwritten.**
---remove-unused-components | boolean | Remove unused components from the `bundle` output.
---skip-decorator | [string] | Ignore certain decorators. See the [Skip preprocessor, rule, or decorator section](#skip-preprocessor-rule-or-decorator).
---skip-preprocessor | [string] | Ignore certain preprocessors. See the [Skip preprocessor, rule, or decorator section](#skip-preprocessor-rule-or-decorator).
---skip-rule | [string] | Ignore certain rules. See the [Skip preprocessor, rule, or decorator section](#skip-preprocessor-rule-or-decorator).
---version | boolean | Show version number.
-
+| Option                     | Type     | Description                                                                                                                                                                                                                                                     |
+| -------------------------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| apis                       | [string] | List of API root definition filenames or names assigned in the `apis` section of your Redocly configuration file. Default values are all names defined in the `apis` section within your configuration file.                                                    |
```

Comment:
Should the values be comma separated?

Reply:
I think they shouldn't. The most common behaviour for positional arrays is simply separating items by spaces.

Reply:
Agree that there should not be commas in CLI inputs. It's more expected to do repeated arguments

Reply:
This is a positional argument (not a named parameter). I suppose it is unlikely anybody will use the names of other options as their API names.